RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronIDProducer.cc

```diff
@@ -124,7 +119,23 @@ void LowPtGsfElectronIDProducer::produce(edm::StreamID, edm::Event& event, const
     if (gsf.isNull()) {
       continue;
     }
-    float unbiased = (*unbiasedH)[gsf];
+
+    // Attempt to obtained ElectronSeed unbiased BDT score
+    float unbiased = -10.;
+    if (unbiasedH.isValid()) {
+      unbiased = (*unbiasedH)[gsf];
+    } else {
+      pat::Electron const* casted = dynamic_cast<pat::Electron const*>(ele.get());
+      if (casted == nullptr) {
```

Comment:
is this branch really still needed?
I see some vestige in PhysicsTools/PatAlgos/python/slimming/slimmedLowPtElectrons_cff.py, but that's commented out and is unclear if it was a result of a dead-end debugging

Reply:
Removed.